Recognise OpenBSD to unbreak build:
  error[E0425]: cannot find value `get_shell_ffi` in this scope
    --> src/utils.rs:69:48
https://github.com/Spotifyd/spotifyd/pull/1253

Index: src/utils.rs
--- src/utils.rs.orig
+++ src/utils.rs
@@ -1,7 +1,7 @@
 use log::trace;
 use std::env;
 
-#[cfg(any(target_os = "freebsd", target_os = "linux"))]
+#[cfg(any(target_os = "freebsd", target_os = "linux", target_os = "openbsd"))]
 fn get_shell_ffi() -> Option<String> {
     use libc::{geteuid, getpwuid_r};
     use std::{ffi::CStr, mem, ptr};
